src/migrateSettingsFolder.ts

```diff
@@ -21,7 +21,9 @@ const emptySettingsFolders = _pick(
 function migrateSettingsMap(legacySettingsMap: {
   [settingKey: string]: any;
 }): Partial<Settings> {
-  const migratedSettingsMap: Partial<Settings> = {};
+  const migratedSettingsMap: Partial<
+    Settings & { "userFilters.areEnabled"?: boolean }
```

Comment:
`userFilters.areEnabled` has been replaced by the `canUseUserFilters` permission in 5.10.
It was deprecated [here](https://github.com/activeviam/activeui/pull/2653) and removed from the `Settings` type [here](https://github.com/activeviam/activeui/pull/2685).

Reply:
We'll need to import types from different versions of the SDK to be precise: here, it should be the settings of 5.0.
It seems like we should be able to do it with aliases: https://dev.to/3imed_jaberi/how-to-install-multiple-versions-of-a-npm-package-at-the-same-time-project-2a29.
Could you try it out ?